### PR TITLE
Added DAO Engineer JD

### PR DESCRIPTION
**Pull request by idiom-bytes**
Opening up PR for DAO Engineer JD

### CODE REVIEW

File: dao.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# 🦞 Senior Blockchain Developer - OceanDAO
+[Ocean Protocol](https://www.oceanprotocol.com/) has a vibrant, expanding ecosystem of engineers, data scientists, neuroscientists, ambassadors, podcasters, that are build traction across 5 pillars of growth, receiving grants and funding to deliver results. To serve them better, we’re hiring!
+
+[OceanDAO](https://www.oceanprotocol.com/dao) is a key component that brings decentralization and scale to participants that are driving traction into the Ocean ecosystem. We’re growing the OceanDAO team. If you love to work with crypto, to build and to engage deeply with an enthusiastic community, we want to hear from you.
```
Comment (LoznianuAnamaria) [inline dao.md]: If you love to work with crypto, to build and **_to_** engage.... remove the 'to' in front of engage

File: dao.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+
+## Responsibilities
+* You will design & build for OceanDAO core software, from the level of blockchain to the level of dapps. Integrating governance, treasury management, viral mechanisms, and whatever else is needed for OceanDAO to be concretized.
+* The scope of the software is well defined but may evolve over time as DAOs are nascent, e.g. integrating decentralized courts, and decentralizing funding mechanisms. You will co-plan and co-design the product with the OceanDAO team in an agile, open-source development process.
+
+
+## Minimum Qualifications
+* 6+ months of Solidity experience
+* 3+ years backend experience, including NodeJS
```
Comment (LoznianuAnamaria) [inline dao.md]: I would order the experience needed based on years, in reverse order. To 4+ years of software experience to be the first one, then 3+ years of backend the second one and the 6+ months, the last one

File: dao.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+
+## Nice-to-haves
+* 2+ years working with cloud infrastructure, supporting devops.
+* 1+ years CI/CD experience (e.g. Travis, tox)
+* You’re obsessed with blockchain and DAOs. You play with dapps, vote on proposals, or have contributed code to a DAO.
+* Excellent writing and communication skills. Candidate will engage with the community, receive feedback, identify usage patterns, and communicate macro outcomes.
+* Successfully delivered on promises, comfortable working autonomously and within a team
+* Differentiate good and bad feedback to find real pain points to improve product/program based on learnings and manage and update scripts and reward function accordingly
+* Proven experience in building meaningful technical communities around projects and ability to build and manage relationship with a key stakeholders
```
Comment (LoznianuAnamaria) [inline dao.md]: * Manage relationships(plural)
* With key stakeholders, remove the 'a' key stakeholders

File: dao.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+* 4+ years of software experience
+* Experience working in technical/development roles in the crypto projects
+* Process-driven, proactive - able to report on your activity and communicate well with the team and DAO (community).
+
+
+## Nice-to-haves
+* 2+ years working with cloud infrastructure, supporting devops.
+* 1+ years CI/CD experience (e.g. Travis, tox)
+* You’re obsessed with blockchain and DAOs. You play with dapps, vote on proposals, or have contributed code to a DAO.
+* Excellent writing and communication skills. Candidate will engage with the community, receive feedback, identify usage patterns, and communicate macro outcomes.
```
Comment (LoznianuAnamaria) [inline dao.md]: Instead of 'Candidate', I would make this personal. You will engage with the community...

**Review by LoznianuAnamaria — CHANGES_REQUESTED** [review]

**Comment by idiom-bytes** [comment]
Ok, I've updated the description based on your recommendations. Thank you!

@trentmc I have adjusted the technical expectations to focus more on web3/backend/infra. This reflects the intention on separating backend/frontend initiatives, in addition to decentralizing more frontend stuff (i.e. Uni Labs vs. Uni FE debacle). 

Moreover, Ocean whitepaper and looking out further into the future (beyond 2022) may bring more onchain systems, and perhaps even para-chain implementations of voting/funding/etc to make these systems low-cost & truly web3/decentralized (vs. off-chain gasless voting on proposals via snapshot.... which is same same, but different). 

Please LMK if this checks out with your intuitions, or if you want to put some emphasis on full-stack.

File: dao.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+## Responsibilities
+* You will design & build for OceanDAO core software, from the level of blockchain to the level of dapps. Integrating governance, treasury management, viral mechanisms, and whatever else is needed for OceanDAO to be concretized.
+* The scope of the software is well defined but may evolve over time as DAOs are nascent, e.g. integrating decentralized courts, and decentralizing funding mechanisms. You will co-plan and co-design the product with the OceanDAO team in an agile, open-source development process.
+
+
+## Minimum Qualifications
+* 4+ years of software experience
+* 3+ years backend experience, including NodeJS
+* 6+ months of Solidity experience
```
Comment (trentmc) [inline dao.md:15]: Also 1+ years web3.js

File: dao.md [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+* Stellar research skills
+* Preferred work hours: 9-5 on North American time zones. We start early to sync with the EU team (7am PDT).
+* We don’t expect you to be an expert in all of these. Rather, we’d love to see that you’re interested in them, perhaps know one or two as a starting point, and most importantly, that you’ll learn more about them as you grow with us professionally and personally.
+
+
+
+## Your Next Step?
+If you love to build things and care deeply about the future of data, we want to hear from you! Please email [devjobs@oceanprotocol.com](mailto:devjobs@oceanprotocol.com), with:
+* your cover letter - why you’d like to join
+* your CV
```
Comment (trentmc) [inline dao.md:37]: Also ask for eth address(es) or ens names. Call it optional. This will be super useful for us though!

**Review by trentmc — APPROVED** [review]

**Comment by trentmc** [comment]
Overall I'm fine with it.

**Review by LoznianuAnamaria — APPROVED** [review]

**Pull request merged**